### ISSUE TITLE
Buy a can crusher for the kitchen

### ISSUE THREAD

**Issue by ajlennon** [DoESLiverpool/somebody-should#1355]
Should we get a can crusher to put on the wall in the kitchen for cans going into the recycling box?

e.g. https://www.amazon.co.uk/16oz-Can-Crusher-Crush-500ml/dp/B0069K0FKU

**Comment by goatchurchprime** [comment]
Nope!   https://lifehacker.com/dont-crush-cans-before-recycling-them-1833374490

**Comment by ajlennon** [comment]
If you read the article it says this is only an issue with mixed recycling. We should understand first if a bag of crushed cans is treated as mixed recycling.

**Comment by Sean-anotherone** [comment]
Why buy a can crusher when we could build one......

https://sciencedemonstrations.fas.harvard.edu/presentations/can-crusher-magnetic-implosion

**Comment by ajlennon** [comment]
I agree that’s a rather wonderful idea but it does seem more of a can thinner rather than a can crusher.

Now if you could use your contacts at CERN/LHC to source us a small captured black hole I think that could be far more fun to play with?
